model/src/main/java/org/cbioportal/model/CancerStudy.java

```diff
@@ -214,4 +214,4 @@ public Integer getCompleteSampleCount() {
     public void setCompleteSampleCount(Integer completeSampleCount) {
         this.completeSampleCount = completeSampleCount;
     }
-}
```

Comment:
can we git checkout the file instead? kind of confusing.

Reply:
@khzhu done